mmaction/datasets/pipelines/loading.py

```diff
@@ -901,7 +901,7 @@ def __call__(self, results):
 
 @PIPELINES.register_module()
 class OpenCVInit:
-    """Using OpenCV to initalize the video_reader.
+    """Using OpenCV to initialize the video_reader.
```

Comment:
😕 This typo is been fixed many times...

Reply:
Got it! a stupid typo may occur in many places. It is a better strategy to global check, when finding a typo word.